front/app/hooks/useInputSchema.ts

```diff
@@ -21,10 +21,755 @@ export default (projectId: string | undefined) => {
     const observable = ideaJsonFormsSchemaStream(projectId).observable;
 
     const subscription = observable.subscribe((response) => {
+      // TODO Remove this. Temporary fo testing.
+      const dummyAPIResponse = {
+        json_schema_multiloc: {
```

Comment:
Added a temp schema, since the back office form building isn't available in this branch yet.

Reply:
No difference - you're correct. There just isn't a way to build a custom form in the back office yet (that's on Edwin's branch). So when the the custom form is retrieved from the endpoint, it's empty since there is no custom form. I've added a some dummy data for now so I can see a survey form :)

Reply:
> There just isn't a way to build a custom form in the back office yet

Correct, but you can do it in the Rails console. Here is an example. You can add as many fields as you like.
```
docker-compose run --rm web rake cl2back:add_extra_field['localhost','<PROJECT_ID HERE>','Swimming pool name','Propose a name for the new swimming pool','text','false']
```

Reply:
@koendehondt Is it possible this command is different on linux CLI?